docs/docs/reference/erased-terms.md

```diff
@@ -164,11 +165,15 @@ def h(x: erased Int => Int) = ...
 class K(erased x: Int) { ... }
 ```
 
+
```

Comment:
Is the extra line _before_ also required?

Reply:
Nope, its not but I figured it reads better in the raw version . However, the fix @allanrenucci provides is better IMO